### PR TITLE
onboarding add summary api

### PR DESCRIPTION
**Pull request by yravinderkumar33**

### CODE REVIEW

File: ansible/roles/stack-sunbird/templates/sunbird_report-service.env [hunk 1]
```diff
@@ -6,4 +6,5 @@ SUNBIRD_REPORTS_DB_USER={{dp_postgres_username}}
 SUNBIRD_SERVER_PORT=3030 
 SUNBIRD_BASE_REPORT_URL=report
 SUNBIRD_REPORTS_TABLE_NAME=report
+SUNBIRD_REPORT_SUMMARY_TABLE_NAME=report_summary
```
Comment (SMYALTAMASH) [inline ansible/roles/stack-sunbird/templates/sunbird_report-service.env:9]: will this value be always constant?

Comment (yravinderkumar33) [reply ansible/roles/stack-sunbird/templates/sunbird_report-service.env:9]: both report table and report summary table are hard-coded for now.

Comment (SMYALTAMASH) [reply ansible/roles/stack-sunbird/templates/sunbird_report-service.env:9]: declare them in defaults and have it as a variable

**Comment by SMYALTAMASH** [comment]
can be merged now

**Pull request merged**